### PR TITLE
Init

### PR DESCRIPTION
**Pull request by conrad760**
This is just the start of the project but it is probably a good time to start thinking about requirements for a pull request. I am going to say a good rule of thumb for now is you can't approve your own. BUT we should start to think about CI and Unit test here soon.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+					"id": &graphql.ArgumentConfig{
+						Type: graphql.String,
+					},
+				},
+				Resolve: func(p graphql.ResolveParams) (interface{}, error) {
+					idQuery, isOK := p.Args["id"].(string)
+					if isOK {
+						return data[idQuery], nil
+					}
+					return nil, nil
```
Comment (xylafur) [inline main.go:60]: Shouldn't we return an error here?

File: main.go [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+	},
+)
+
+func executeQuery(query string, schema graphql.Schema) *graphql.Result {
+	result := graphql.Do(graphql.Params{
+		Schema:        schema,
+		RequestString: query,
+	})
+	if len(result.Errors) > 0 {
+		fmt.Printf("wrong result, unexpected errors: %v", result.Errors)
```
Comment (xylafur) [inline main.go:78]: Should return an error here too?

Comment (xylafur) [reply main.go:78]: Honestly not sure on go standards regarding that

File: main.go [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+		fmt.Print("Error:", err)
+		isOK = false
+	}
+	err = json.Unmarshal(content, result)
+	if err != nil {
+		isOK = false
+		fmt.Print("Error:", err)
+	}
+	return
+}
```
Comment (xylafur) [inline main.go]: Needs to return isOK

**Review by xylafur — CHANGES_REQUESTED** [review]

File: main.go (hunk 2 above)
Comment (conrad760) [reply main.go:78]: I think this is fine until we build a logger

File: main.go (hunk 1 above)
Comment (conrad760) [reply main.go:60]: no ;)

**Review by conrad760 — COMMENTED** [review]
review

**Comment by xylafur** [comment]
Looks good

**Pull request merged**